src/plugins/formvalid/formvalid-en.hbs

```diff
@@ -456,6 +456,91 @@
 	</div>
 </section>
 <hr>
+<section>
+	<h2>Examples using required fields without asterisks</h2>
```

Comment:
@Garneauma @duboisp Should a paragraph be added to this section (and perhaps even the docs) to explicitly state that the ``required-no-asterisk`` class is an alternate way of styling required fields? Or is the status quo good enough to get the point across?

Reply:
Yes, please.

Reply:
Done :)!